app/src/components/googleMap.tsx

```diff
@@ -507,17 +535,18 @@ class SimpleMap extends React.Component<MapProp, MapState> {
         if (currentMode.searchArea.se.lng < -180) {
             currentMode.searchArea.se.lng += 360;
         }
-
+        
```

Comment:
please, don't add empty spaces all over the file!

Reply:
this is done by WebStrom, I couldn't control

Reply:
please setup your WebStorm then

Reply:
which setting I need to change?

Reply:
google it, idk

Reply:
okay

Reply:
no luck yet?